File: app/src/main/res/ui/resetPassword/ResetPasswordActivity.java
package com.khedr_and_basuoin.deliveryapp.ui.resetPassword;

import androidx.appcompat.app.AppCompatActivity;
import androidx.databinding.DataBindingUtil;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.khedr_and_basuoin.deliveryapp.R;
import com.khedr_and_basuoin.deliveryapp.databinding.ActivityResetPasswordBinding;
import com.khedr_and_basuoin.deliveryapp.ui.verifyCode.VerifyCodeActivity;

public class ResetPasswordActivity extends AppCompatActivity implements View.OnClickListener {
    ActivityResetPasswordBinding b;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        b= DataBindingUtil.setContentView(this,R.layout.activity_reset_password);
        b.btSendCodeReset.setOnClickListener(this);

    }

    @Override
    public void onClick(View v) {
        if (v==b.btSendCodeReset){
            startActivity(new Intent(this, VerifyCodeActivity.class));
        }
    }
}

File: app/src/main/java/com/khedrandbasuoni/baytifood/ui/activities/resetPassword/ResetPasswordActivity.kt
package com.khedrandbasuoni.baytifood.ui.activities.resetPassword

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.khedrandbasuoni.baytifood.R
import android.content.Intent
import android.view.View
import com.khedrandbasuoni.baytifood.databinding.ActivityResetPasswordBinding
import com.khedrandbasuoni.baytifood.ui.activities.verifyCode.VerifyCodeActivity

class ResetPasswordActivity : AppCompatActivity(), View.OnClickListener {
    var b: ActivityResetPasswordBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        b = DataBindingUtil.setContentView(this, R.layout.activity_reset_password)
        b!!.btSendCodeReset.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        if (v === b!!.btSendCodeReset) {
            startActivity(Intent(this, VerifyCodeActivity::class.java))
        }
    }
}